docs/design/job_architecture.md

```diff
@@ -18,13 +18,11 @@ The narrative backend uses the JobComm module to send and receive messages from
 The front- and backend have a shared vocabulary of message types and parameters, loaded from the file `kbase-extension/static/kbase/config/job_config.json`. In the frontend, the message param and type names are stored in the JobCommMessages object exported by `jobCommChannel.js`:
 
 ```js
-define(['common/jobCommChannel'], (JobComms) => {
+define(['common/jobCommMessages'], (JobCommMessages) => {

```

Comment:
Throughout the PR, imports of `jobCommChannel` have been switched for importing the `jobCommMessages` module instead.